cart/views.py

```diff
@@ -1,3 +1,71 @@
-from django.shortcuts import render
+import json, bcrypt
 
-# Create your views here.
+from django.views     import View
+from django.http      import JsonResponse
+
+from .models   import Cart, Ingredient, User
+
+from utils import login_decorator
+
+class CartListView(View):
+    @login_decorator
+    def post(self,request):
+        try:
+            data = json.loads(request.body)
+            user = request.user
+
+            if data['count'] <= 0:
+                return JsonResponse({"message": "LESS_THAN_MINIMUM_QUANTITY"}, status=400)
+
+            duplicate_user = Cart.objects.filter(user_id=user.id)
+
+            for cart in duplicate_user :
+                if cart.ingredient_id == data['ingredient_id'] :
+                    cart.count += data['count']
+                    cart.save()
+                    return JsonResponse({'message':'ADDITIONAL_SUCCESS'}, status=200)
+                    
+            Cart.objects.create(
+                count         = data['count'],
+                ingredient_id = data['ingredient_id'],
+                user          = user
+                )
+            return JsonResponse({'message':'SUCCESS'}, status=201)
+        
+        except KeyError:
+            return JsonResponse({"message": "KEY_ERROR"}, status=400)
+        
+        except Cart.DoesNotExist:
+          return JsonResponse({'message': 'THIS ACCOUNT DOES NOT EXIST'}, status=400)
+    
+    @login_decorator
+    def get(self,request):
+        try:
+            user = request.user
+            
+            users_ingredient_list = Cart.objects.filter(user_id=user)
+            cart_list = []
+
```

Comment:
위에서 드린 피드백과 같습니다.
추가로, `user = request.user`의 경우 아래에서 계속 사용되는게 아니라고 하면, 굳이 변수로 만들 필요가 없어 보입니다.
아래와 같이 필요한 곳에서 바로 사용하면 좋을 것 같습니다!

```suggestion
            carts = Cart.objects.filter(user_id=request.user)
            
            for cart in carts :
                ...
```

Reply:
post부분은 는 3개 이상이라  변수에 저장하였고 get 부분에서는 한번밖에 사용이 안되어 바로 사용할 수 있도록 수정하였습니다!